CONST_Makefile

```diff
@@ -1144,24 +1141,6 @@ ifneq ($(TOMCAT_START_COMMAND),)
 endif
 endif # !DOCKER
 
-print/WEB-INF/lib/jasperreports-functions-$(JASPERREPORTS_VERSION).jar:
-	$(PRERULE_CMD)
-	mkdir -p $(dir $@)
-	curl --location --output $@ https://kent.dl.sourceforge.net/project/jasperreports/jasperreports/JasperReports%20$(JASPERREPORTS_VERSION)/jasperreports-functions-$(JASPERREPORTS_VERSION).jar
-	unzip -t -q $@
-
-print/WEB-INF/lib/joda-time-1.6.jar:
-	$(PRERULE_CMD)
-	mkdir -p $(dir $@)
-	curl --max-redirs 0 --location --output $@ http://central.maven.org/maven2/joda-time/joda-time/1.6/joda-time-1.6.jar
-	unzip -t -q $@
-
-print/WEB-INF/lib/postgresql-9.3-1102.jdbc41.jar:
-	$(PRERULE_CMD)
-	mkdir -p $(dir $@)
-	curl --max-redirs 0 --location --output $@ https://jdbc.postgresql.org/download/postgresql-9.3-1102.jdbc41.jar
-	unzip -t -q $@
-
 $(VENV_BIN)/generate_controller$(PYTHON_BIN_POSTFIX): .build/dev-requirements.timestamp
 
 # Tile cloud chain
```

Comment:
You are not supposed to change the files starting by `CONST_` This needs to be fixed in c2cgeoportal instead:
https://github.com/camptocamp/c2cgeoportal/blob/master/c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl#L436-L439